### PR TITLE
UCP: Check if the PD server responds with less timestamps

### PR DESCRIPTION
**Pull request by Renkai**
resolved https://github.com/tikv/client-rust/issues/95.

### CODE REVIEW

File: src/pd/timestamp.rs [hunk 1]
```diff
@@ -132,10 +137,12 @@ impl<'a> Stream for TsoRequestStream<'a> {
         let pending_requests = self.pending_requests.clone();
         let mut pending_requests = pending_requests.borrow_mut();
         let mut count = 0;
```
Comment (sticnarf) [inline src/pd/timestamp.rs]: It looks that `count` is redundant as we can rely on `requests.len()`.

Comment (Renkai) [reply src/pd/timestamp.rs]: > It looks that `count` is redundant as we can rely on `requests.len()`.

updated

Comment (sticnarf) [reply src/pd/timestamp.rs]: Please fix clippy

**Comment by sticnarf** [comment]
Thanks for your contribution!

**Comment by you06** [comment]
@Renkai the points is rewarded. FYI, all points in tikv org is shared, which means you need to pick up some medium/hard tasks next time :)

**Pull request merged**